### DISCUSSION TITLE
"date-fns". The package may have incorrect main/module/exports

### DISCUSSION THREAD

**Discussion by seydouromeo** [js-fns/date-fns#3357] [General] [answered]
Hi there ! can anyone help me with this

Failed to resolve entry for package "date-fns". The package may have incorrect main/module/exports specified in its package.json.
06:47:29 [vite] Internal server error: Failed to resolve entry for package "date-fns". The package may have incorrect main/module/exports specified in its package.json.
  Plugin: vite:import-analysis

**Comment by Meriemi-git** [comment]
Same problem

**Comment by KaytrosGechoArka** [comment]
Me too! help us

**Comment by moisesbites** [comment]
the same error, with sveltekit and  vite.

**Comment by kossnocorp** [comment, marked as answer]
Please anyone who has this problem see the issue where you'll find further instructions: https://github.com/date-fns/date-fns/issues/3624
